### PR TITLE
[Port] Many Traitor Ports

### PR DESCRIPTION
**Pull request by Profakos**
We begun a grand port project with Collen and the others!
This one ports:
- TC price rebalance.
- Sleepy Pen replacement
- C4 changes
- An unused var override removal
- Adds smuggler's satchel
- Adds maroon and destroy. Marron is 30% chance to replace traitor and ling assassinate. The target doesn't have to be dead, just not on Centcomm. Destroy is about killing the AI. I did not make it available randomly, admins can still force it.

Still testing, and checking if there are other related PRs that can be merged at the moment.

### CODE REVIEW

**Comment by Profakos** [comment]
Removed Syndie briefcase with money, because of a weird bug I don't understand. If it spawns in your hand, and you don't put it down and pick it up again, it won't display its contents until you do so.
It works fine in every other case.

**Comment by Profakos** [comment]
I am really sad and annoyed by the bug with the money briefcase :( But otherwise the port is ready.

**Comment by SuperSayu** [comment]
Logfiles snuck into the PR, please remove them

**Comment by Profakos** [comment]
Logfiles removed. I am sorry I didn't notice them ;-;

**Comment by Profakos** [comment]
One last add, a syndie headset. Increased price to 11 from 5 on community feedback.

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -511,3 +549,35 @@ var/list/uplink_items = list()
 		U.uses -= max(0, I.cost)
 		feedback_add_details("traitor_uplink_items_bought","RN")
 		return new I.item(loc)
+
+/datum/uplink_item/badass/surplus_crate
+	name = "Syndicate Surplus Crate"
+	desc = "A crate containing 50 telecrystals worth of random syndicate leftovers."
+	cost = 20
+	item = /obj/item/weapon/storage/box/syndicate
+	excludefrom = list(/datum/game_mode/nuclear)
```
Comment (SuperSayu) [inline code/datums/uplink_item.dm:558]: If you can't get the crate during nuclear games, when else will you have 20 telecrystals?  I am confused

EDIT: Right, tc rebalance, my bad

**Comment by Profakos** [comment]
Absolutely final things for this PR.

**Pull request merged**